src/views/index.html

```diff
@@ -33,7 +33,7 @@ <h2 id="splash-hack-cambridge-wordmark">Hack<br>Cambridge<br><span class="pixel"
                                 <div class="five unit column">
                                     <div id="splash-intro-text">
                                         <p>
-                                            In January 2018, Cambridge University’s annual hackathon will yet again bring together 300 hackers, programmers, designers and more from universities all over the world.<br>For 24 hours they will build, break and innovate to produce projects that push the boundaries of technology.
+                                            From the 20<sup>th</sup> to the 21<sup>st</sup> of January 2018, Cambridge University’s annual hackathon will yet again bring together 300 hackers, programmers, designers and more from universities all over the world.<br>For 24 hours they will build, break and innovate to produce projects that push the boundaries of technology.
```

Comment:
Would be ideal to get these from `js/shared/dates.js` — would you like to do that in this pull or shall we create an issue to do it later?

Reply:
Create an issue and do it later. :+1: